src/main/java/core/basesyntax/dao/ma/MentorDaoImpl.java

```diff
@@ -11,6 +13,13 @@ public MentorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public List<Mentor> findByAgeGreaterThan(int age) {
-        return null;
+        try (Session session = sessionFactory.openSession()) {
+            Query<Mentor> query = session.createQuery("FROM Mentor m "
+                    + "WHERE m.age >= :age", Mentor.class);
```

Comment:
age "greaterThan", not "equalOrGreaterThan"

Reply:
done ✔️